tests/integration/template_complete_run.cfg

```diff
@@ -13,7 +13,7 @@ www = "#expand user_www#zppy_test_complete_run_www"
 [climo]
 active = True
 walltime = "00:30:00"
-years = "1850:1854:2", "1850:1854:4",
+years = "1850:1854:2", "1850:1854:4"
```

Comment:
Launches the following:
```
climo_atm_monthly_180x360_aave_1850-1851
climo_atm_monthly_180x360_aave_1852-1853
climo_atm_monthly_180x360_aave_1850-1853
climo_atm_monthly_diurnal_8xdaily_180x360_aave_1850-1851
climo_atm_monthly_diurnal_8xdaily_180x360_aave_1852-1853
climo_atm_monthly_diurnal_8xdaily_180x360_aave_1850-1853
```
Both the `"1850:1854:2"` (2-year increment) jobs and the `"1850:1854:4"` (4-year increment) jobs are launched successfully. Does this mean the trailing comma is no longer a requirement of `string_list` parameters?